manifests/n/NetSarangComputer,Inc/Xshell8/8.0.0069/NetSarangComputer,Inc.Xshell8.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.14.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: NetSarangComputer,Inc.Xshell8
+PackageVersion: 8.0.0069
+InstallerType: exe
+InstallerSwitches:
+  Silent: Y
+  SilentWithProgress: Y

```

Comment:
These switches don't seem to work
![image](https://github.com/user-attachments/assets/2f0758a7-63fe-43ec-869c-75e8273348f3)